crates/swc_plugin/src/lib.rs

```diff
@@ -1,39 +1,31 @@
-/// **Don't use this**
-#[doc(hidden)]
-pub extern crate abi_stable;
-/// **Don't use this**
-#[doc(hidden)]
-pub extern crate swc_common;
-/// Fix malloc.
-extern crate swc_node_base;
+// Reexports
+pub use swc_common::chain;
+pub use swc_ecma_ast::*;
+pub use swc_ecma_visit::*;
 
-use abi_stable::std_types::{RResult, RStr, RString};
-use anyhow::Context;
-use serde::de::DeserializeOwned;
-/// Reexported for convenience.
-pub use swc_plugin_js_api::*;
+#[cfg(target_arch = "wasm32")]
+pub mod memory {
```

Comment:
I think it's better to have `#[doc(hidden)]`.

Reply:
I'm somehow seeing weird invalid memory reference err when running `test -p lib`, so guarded with target arch instead. I'll dig around memoryview's page instead for the future PRs.

Reply:
I mean additional `#[doc(hidden)]` on the module.

Reply:
Got it, thanks for the clarification.